converter/converters/koki_service_to_kube_v1_service.go

```diff
@@ -0,0 +1,196 @@
+package converters
+
+import (
+	"fmt"
+	"reflect"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/koki/short/types"
+	"github.com/koki/short/util"
+)
+
+func Convert_Koki_Service_To_Kube_v1_Service(service *types.ServiceWrapper) (*v1.Service, error) {
+	var err error
+	kubeService := &v1.Service{}
+	kokiService := &service.Service
+
+	kubeService.Name = kokiService.Name
+	kubeService.Namespace = kokiService.Namespace
+	kubeService.APIVersion = kokiService.Version
+	kubeService.Kind = "Service"
+	kubeService.ClusterName = kokiService.Cluster
+	kubeService.Labels = kokiService.Labels
+	kubeService.Annotations = kokiService.Annotations
+
+	if len(kokiService.ExternalName) > 0 {
+		kubeService.Spec.Type = v1.ServiceTypeExternalName
+		kubeService.Spec.ExternalName = kokiService.ExternalName
+		return kubeService, nil
+	}
+
+	kubeService.Spec.Type = v1.ServiceTypeClusterIP
+
+	kubeService.Spec.Selector = kokiService.PodLabels
+	kubeService.Spec.ClusterIP = string(kokiService.ClusterIP)
+	kubeService.Spec.ExternalIPs = revertExternalIPs(kokiService.ExternalIPs)
+	err = revertSessionAffinityInto(kokiService.ClientIPAffinity, &kubeService.Spec)
```

Comment:
Why are some methods using this `revert$TypeInto` pattern while others are using the `revert$Type` pattern?

Reply:
I'll add a comment about this.

The revertX methods return the kube-native value while the revertXInto methods insert values into an existing kube-native value. I use 'Into' if the revert method affects multiple subfields at once.

Reply:
I figured that's what was happening